src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -85,7 +85,8 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
 
         foreach ($lines as &$line) {
             $line = sprintf('%s%s', $prefix, $line);
-            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line));
+            //https://github.com/symfony/symfony/pull/14600
+            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line) + substr_count($line, '\<'));
```

Comment:
Alright. It's done.

In fact I was saying this line was the only line required in our case. But, indeed, that's not true. 
I confirm this is needed, if using tags related to a style in blocks.

First my suggestion, second yours:
![screenshot 2015-05-13 a 00 30 48](https://cloud.githubusercontent.com/assets/2211145/7600037/e861bc6c-f907-11e4-978b-363bc1eef7f5.PNG)


Thank you !